src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const NewRobot = {};
+
+  for (const key in robot) {
+    NewRobot[robot[key]] = key;
+  }
+
+  if (Object.keys(NewRobot).length !== Object.values(robot).length) {
+    return null;
+  }
```

Comment:
You can make it bit simpler
```suggestion
  for (const key in robot) {
    if (robot[key] in newRobot) {
      return null
    }
  
    newRobot[robot[key]] = key;
  }
```

Reply:
ok, thanks